tests/debugger/bug01335.inc

```diff
@@ -36,4 +36,4 @@ class C
 
 $b = new B();
 
-$x = 1; // Breakpoint me, and inspect $b->*TestA\TestB\TestC\A*data1->items
\ No newline at end of file
+$x = 1; // Breakpoint me, and inspect $b->*TestA\TestB\TestC\A*data1->items
```

Comment:
This seems an unrelated change, and hence don't add it to the commits in this PR.

Reply:
I opened https://github.com/xdebug/xdebug/pull/605 for it, I will then rebase this PR